kube-shell/build-and-push.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -e
+
+: ${REPO:=portainer/kubectl-shell}
+: ${KUBERNETES_RELEASE:=v1.21.1}
+: ${TAG:=latest}
+
+docker_image_build_and_push()
+{
+  docker buildx build -o type=docker \
```

Comment:
A small suggestion here. Setting the output type to be `registry` pushes the image soon as the build finishes. 
This means a separate line of `docker image push` is  _not_ required anymore:

```sh
--output=type=registry
```

This is useful in case you need a Windows image to be built :)

Reply:
have some docker create manifest error if we use --output type=registry or --push, so stick with -o type=docker and docker push for now, at least this is working version.